index.html

```diff
@@ -13,7 +13,112 @@
 </head>
 
 <body>
+    <nav class="navbar">
+        <div class="navbar-title">
+            Astra
+        </div>
+        <div class="navbar-items">
+            <div>
+                <ul class="navbar-list">
+                    <li class="navbar-item navbar-icon">
+                        <a href="cart.html" class="navbar-link">
+                            <span class="btn-icon">
+                                <i class="fas fa-shopping-cart"></i>
+                            </span>
+                        </a>
+                    </li>
+                    <li class="navbar-item navbar-icon">
+                        <a href="wishlist.html" class="navbar-link">
+                            <i class="fas fa-heart"></i></a>
+                    </li>
+                </ul>
+            </div>
+            <a href="login.html" class="btn btn-primary">Login</a>
+        </div>
+    </nav>
+    <div class="container">
+        <div class="grid grid-1-4">
+            <div class="product-type product-type-1">
+                <a href="product.html"><span class="product-name">Men</span></a>
+            </div>
```

Comment:
suggestion: you should have a tag outside and then enclose the product name in it, the reason being when we take our cursor to that type, it would be nice to click on the card instead of that particular name. This is just my observation, you can make the changes later.

Reply:
yeah sure